CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 
 #### Patch Changes
 
+- Remove duplicate database indices if they exist
```

Comment:
This is the changelog for the product user.
Rather explain how the change make the product better for them: saves space? better performance?